appconfig/settings/wdb.q

```diff
@@ -2,4 +2,3 @@
 \d .wdb
 replay:0b;     // We don't want to replay as the wdb will start before the period it is meant to subscribe to
 upd:{[t;x]};   //Discard data until the end of period signals the start of new period
-
```

Comment:
remove unrelated whitespace change please

Reply:
should be fixed now